doc/building_ios.md

```diff
@@ -0,0 +1,47 @@
+# Building the JDK targeting iOS
+
+> [!NOTE]
+> Follow the instructions in [building.md](building.md) to make sure you can build the JDK
+> targeting a desktop platform before trying to build for iOS
```

Comment:
instead of "a desktop platform" it is probably better to mention the mac platform explicitly, as we want to make it clear that you (currently) can not build the iOS SDK from a windows or linux system.